keel-web/src/main/kotlin/com/netflix/spinnaker/keel/plugins/KeelServiceSdk.kt

```diff
@@ -0,0 +1,12 @@
+package com.netflix.spinnaker.keel.plugins
+
+import com.netflix.spinnaker.keel.api.actuation.TaskLauncher
+import com.netflix.spinnaker.keel.persistence.KeelRepository
+
+/**
+ * A simple SDK that can be consumed by external plugins to access core Keel functionality.
+ */
+interface KeelServiceSdk {
```

Comment:
@robzienert I just realized this needs to go in `keel-api` to be accessible to plugins, right?

Reply:
@robfletcher With that, I'll need to move `KeelRepository` from `keel-core` to `keel-api` as well. It's an interface, but it's got a bunch of default method implementations... 😕 How do you feel about that?

Reply:
Actually, I think I'll define a read-only interface for plugin access.